tema5/src/main/java/ro/sci/carrental/repository/CustomerDAO.java

```diff
@@ -0,0 +1,84 @@
+package ro.sci.carrental.repository;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ro.sci.carrental.domain.car.Car;
+import ro.sci.carrental.domain.car.Category;
+import ro.sci.carrental.domain.car.GearBoxType;
+import ro.sci.carrental.domain.customer.Address;
+import ro.sci.carrental.domain.customer.Customer;
+import ro.sci.carrental.service.*;
+
+import java.sql.*;
+
+/**
+ * Created by Ivett on 11-Aug-17.
+ */
+public class CustomerDAO {
```

Comment:
Repository cu Dao is cam acelasi lucru, ar tre sa faci o alegere intre cele doua ca e confuz ;)